src/Descriptor.ts

```diff
@@ -1,9 +1,9 @@
-import * as lsif from './lsif'
+import * as scip from './scip'
 
-type Descriptor = lsif.lib.codeintel.lsiftyped.Descriptor
-const Descriptor = lsif.lib.codeintel.lsiftyped.Descriptor
-type Suffix = lsif.lib.codeintel.lsiftyped.Descriptor.Suffix
-const Suffix = lsif.lib.codeintel.lsiftyped.Descriptor.Suffix
+type Descriptor = scip.scip.Descriptor
+const Descriptor = scip.scip.Descriptor
+type Suffix = scip.scip.Descriptor.Suffix
+const Suffix = scip.scip.Descriptor.Suffix
```

Comment:
nit: Can we somehow avoid this being `scip.scip`. It's not a big deal, but it looks a little gross lol.

Reply:
I agree it's not nice and I'd +1 if we can solve it somehow. It's not a regression from this PR though, it was gross before as well